models/models.js

```diff
@@ -0,0 +1,123 @@
+var config = require(__dirname + '/../library/config')
+var log = require(__dirname + '/../library/log.js')
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var mongoPort = process.env.MONGO_PORT || config.ports.mongoPort;
+mongoose.connect('mongodb://localhost:' + mongoPort + '/slacker', function(err, res) {
```

Comment:
This should be parameterized so that we can use [MongoHQ](https://bridge.mongohq.com/signup).